exporter/fileexporter/testdata/config.yaml

```diff
@@ -39,3 +39,7 @@ file/format_error:
 file/compression_error:
   path: ./filename.log
   compression: gzip
+
+file/flush_interval_5:
+  path: ./flushed
+  flush_interval: 5
```

Comment:
```suggestion
  flush_interval: 5

```

Reply:
I have added test cases for different units for flush_interval.